### PR TITLE
Create demo outline for integration specialist

### PR DESCRIPTION
**Pull request by mattschweers**

### CODE REVIEW

File: integration/README.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+[PowerPoint Template and Craft Demo Outline](https://github.com/QuickBase/interview-demos/blob/master/QuickBase_CraftDemo_PresentationTemplate.pptx)
+
+## General Description
+
+The purpose of the craft demo is to give us a jumping-off-point for discussion of your approach to solving a problem and of your code. We are interested in learning how you approach this problem and go about solving it.
+
+We expect this project to take experienced developers about two-four hours to complete. We know not everyone has unlimited time to work on interview projects and we do not expect that everyone completes all of the requirements. However, you may be asked how you might have solved a requirement you missed had you been able to work on this project without time constraints. Additionally, if you want to extend the project beyond the listed requirements (e.g., tracking additional data points in the 'Tweet Tracker' app), please feel free to do so as you see fit. Please note that we prefer higher-code quality over additional features if you start to run out of time.
```
Comment (nvanselow) [inline integration/README.md]: nit: "2-4" or "two to four"

File: integration/README.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+Please be prepared to demo your solution in a working development environment on the day of your interview. When you present your craft demo, we will ask you to describe how you arrived at your solution. After that, we will ask for a tour of your code with a focus on how you arrived at the core functionality and any design decisions and/or tradeoffs you made during the process.
+
+## Demo Project Outline
+
+The Quick Base marketing team has a launched a brand-awareness campaign that is running for the next year and you have been tasked with tracking tweets about the company for the length of this engagement. You have been asked specifically to search Twitter for tweets that contain hashtags from a list compiled by our marketing team and store them in a Quick Base app for future investigation by our business analysts. 
+
+## Demo Project Requirements
+
+1. Your code should collect 100 or more tweets from 2018 that include the following hashtags:
```
Comment (nvanselow) [inline integration/README.md]: So we don't have to update it, maybe make the year generic. "100 or more tweets from the current year"

File: integration/README.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+    * #QuickBase
+    * #NoCode
+    * #LowCode
+    * #QBEMPOWER18
+    * #QBCommunitySummit
+2. Collected tweets should be de-duplicated in the Quick Base app. For example, if a tweet contains both the #NoCode and #LowCode hashtags it should only be stored once in the app with both hashtags captured in the 'Hashtag' field.
+3. All fields in the destination Quick Base app should be populated for each tweet.
+4. Your project output should be a service or application that adheres to both Quick Base and Twitter API best practices.
+5. Industry-standard practices for testing, logging and error-handling are expected as time permits.
```
Comment (nvanselow) [inline integration/README.md]: nit: oxford comma. "testing, logging, and error-handling"

File: integration/README.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+
+## FAQ
+
+### **Working with the Quick Base API**
+
+1. Clone the craft demo 'Tweet Tracker' Quick Base application into a free trial account by visiting this link and clicking the 'Try This App' button: https://www.quickbase.com/exchange/tweet-tracker/[app-id-placeholder].
```
Comment (nvanselow) [inline integration/README.md]: If possible, you'll want to have the link with the marketplace id. The app id can change if we  update the app.

File: integration/README.md (hunk 3 above)
Comment (nvanselow) [reply integration/README.md]: What do you think about including some stretch goals? 
E.g.,
- every 5 min, get  the latest tweets and store them in the app

File: integration/README.md [hunk 5]
```diff
@@ -0,0 +1,47 @@
+# Quick Base Craft Demo | Senior Integration Specialist
```
Comment (nvanselow) [inline integration/README.md]: Let's just say "integration specialist" here. We'll probably use the same demo for other levels as well if we have more than one person in this position.

File: integration/README.md [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+
+Use of your favorite libraries, SDKs and frameworks is welcomed but optional. The project outlined below can be completed with or without these tools.
+
+Please be prepared to demo your solution in a working development environment on the day of your interview. When you present your craft demo, we will ask you to describe how you arrived at your solution. After that, we will ask for a tour of your code with a focus on how you arrived at the core functionality and any design decisions and/or tradeoffs you made during the process.
+
+## Demo Project Outline
+
+The Quick Base marketing team has a launched a brand-awareness campaign that is running for the next year and you have been tasked with tracking tweets about the company for the length of this engagement. You have been asked specifically to search Twitter for tweets that contain hashtags from a list compiled by our marketing team and store them in a Quick Base app for future investigation by our business analysts.
```
Comment (nvanselow) [inline integration/README.md:21]: Nice. I like the setup.

File: integration/README.md [hunk 7]
```diff
@@ -0,0 +1,47 @@
...
+
+The Quick Base marketing team has a launched a brand-awareness campaign that is running for the next year and you have been tasked with tracking tweets about the company for the length of this engagement. You have been asked specifically to search Twitter for tweets that contain hashtags from a list compiled by our marketing team and store them in a Quick Base app for future investigation by our business analysts. 
+
+## Demo Project Requirements
+
+1. Your code should collect 100 or more tweets from 2018 that include the following hashtags:
+    * #QuickBase
+    * #NoCode
+    * #LowCode
+    * #QBEMPOWER18
```
Comment (nvanselow) [inline integration/README.md]: Remove the dated one so we don't have to remember to update this every year.

File: integration/README.md (hunk 6 above)
Comment (nvanselow) [reply integration/README.md:21]: Should we be explicit here in the instructions? E.g., something like:

You will need to create a service that will integrate these two APIs. You can write your service in any language or use any framework you prefer. As a data integration specialist at Quick Base, you will likely be working with a code base written in PHP.

File: integration/README.md [hunk 8]
```diff
@@ -0,0 +1,47 @@
...
+Use of your favorite libraries, SDKs and frameworks is welcomed but optional. The project outlined below can be completed with or without these tools.
+
+Please be prepared to demo your solution in a working development environment on the day of your interview. When you present your craft demo, we will ask you to describe how you arrived at your solution. After that, we will ask for a tour of your code with a focus on how you arrived at the core functionality and any design decisions and/or tradeoffs you made during the process.
+
+## Demo Project Outline
+
+The Quick Base marketing team has a launched a brand-awareness campaign that is running for the next year and you have been tasked with tracking tweets about the company for the length of this engagement. You have been asked specifically to search Twitter for tweets that contain hashtags from a list compiled by our marketing team and store them in a Quick Base app for future investigation by our business analysts. 
+
+## Demo Project Requirements
+
```
Comment (nvanselow) [inline integration/README.md:24]: I think step 1 should be to  setup a trial account and quick base app with a reference to your FAQ below.  I think that will help get people started more quickly.

File: integration/README.md [hunk 9]
```diff
@@ -0,0 +1,47 @@
...
+1. Your code should collect 100 or more tweets from 2018 that include the following hashtags:
+    * #QuickBase
+    * #NoCode
+    * #LowCode
+    * #QBEMPOWER18
+    * #QBCommunitySummit
+2. Collected tweets should be de-duplicated in the Quick Base app. For example, if a tweet contains both the #NoCode and #LowCode hashtags it should only be stored once in the app with both hashtags captured in the 'Hashtag' field.
```
Comment (nvanselow) [inline integration/README.md]: nit: Use `1.` for everything so we don't have to keep fixing the numbering when we edit.  Same throughout.

File: integration/README.md (hunk 6 above)
Comment (nvanselow) [reply integration/README.md:21]: I think I quick diagram of the quickbase app or the table(s) could provide some useful context here.

**Review by nvanselow — APPROVED** [review]
Looks good. A few comments, but I think the setup makes sense for this particular role.

**Comment by mattschweers** [comment]
@nvanselow thank you! Making these changes now

File: integration/README.md (hunk 3 above)
Comment (mattschweers) [reply integration/README.md]: We can talk offline and update that, I have some thoughts there

File: integration/README.md (hunk 4 above)
Comment (mattschweers) [reply integration/README.md]: Yep, I said 'app-id' but meant marketplace ID (which we have now!)

File: integration/README.md (hunk 5 above)
Comment (mattschweers) [reply integration/README.md]: true, thanks

File: integration/README.md (hunk 8 above)
Comment (mattschweers) [reply integration/README.md:24]: Copying the app prompts you to set up a trial account ;)

File: integration/README.md (hunk 6 above)
Comment (mattschweers) [reply integration/README.md:21]: Thanks!

I added your comments to the item about project output. I'll talk to you offline about the diagram comment - I don't disagree but want to be careful there

File: integration/README.md (hunk 8 above)
Comment (mattschweers) [reply integration/README.md:24]: Update: except if the app isn't on the marketing site. Followed your advice and tweaked this to include signing up for a trial and our help site content for cloning from exchange for the specific app

File: integration/README.md (hunk 9 above)
Comment (mattschweers) [reply integration/README.md]: good catch!

File: integration/README.md (hunk 7 above)
Comment (mattschweers) [reply integration/README.md]: All set

**Pull request merged**